cilium-cli/cli/connectivity.go

```diff
@@ -191,6 +191,7 @@ func newCmdConnectivityTest(hooks api.Hooks) *cobra.Command {
 	cmd.Flags().StringVar(&params.TestConnDisruptImage, "test-conn-disrupt-image", defaults.ConnectivityCheckImagesTest["ConnectivityTestConnDisruptImage"], "Image path to use for connection disruption tests")
 	cmd.Flags().StringVar(&params.FRRImage, "frr-image", defaults.ConnectivityCheckImagesTest["ConnectivityTestFRRImage"], "Image path to use for FRR")
 	cmd.Flags().StringVar(&params.SocatImage, "socat-image", defaults.ConnectivityCheckImagesTest["ConnectivityTestSocatImage"], "Image path to use for multicast tests")
+	cmd.Flags().StringVar(&params.LoadbalancerL7Image, "loadbalancer-l7-image", defaults.ConnectivityCheckOptionalImagesTest["ConnectivityTestLoadbabancerL7Image"], "Image path to use for L7 Loadbalancer tests")
```

Comment:
Is this for an image of an L7 loadbalancer? Or of a service backend that also can target itself? Can it be used for all L7 LB integration tests, or just a subset of them? Maybe clarify the option name accordingly :-)

Reply:
ah good point, I will update the flag name to specific name for image, we might use it later for other use case :+1: